ios/browser/api/networking/models/BraveCertificateFingerprint.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import <Foundation/Foundation.h>
+
+#ifndef BRAVE_IOS_BROWSER_API_NETWORKING_MODELS_BRAVE_CERTIFICATE_FINGERPRINT_H_
+#define BRAVE_IOS_BROWSER_API_NETWORKING_MODELS_BRAVE_CERTIFICATE_FINGERPRINT_H_
+
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
```

Comment:
why would this ever not be the case (compiling in brave-core) and why would the path be different?